lib/versioning/rez/transform.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  ascendingRange,
+  descendingRange,
+  exactVersion,
+  inclusiveBound,
+  lowerBound,
+  matchVersion,
+  upperBound,
+} from './pattern';
+
+function getVersionParts(input: string): [string, string] {
+  const versionParts = input.split('-');
+  if (versionParts.length === 1) {
+    return [input, ''];
+  }
+
+  return [versionParts[0], '-' + versionParts[1]];
+}
+export function padZeroes(input: string): string {
```

Comment:
```suggestion

export function padZeroes(input: string): string {
```

Please add empty newlines between functions